packages/api/src/lib/firebase.js

```diff
@@ -1,45 +1,5 @@
-const fs = require('fs');
-const path = require('path');
-const firebase = require('firebase');
-const firebaseAdmin = require('firebase-admin');
 const firebaseFunctions = require('firebase-functions');
 
-const config = {
-  apiKey: 'AIzaSyAVl632H0LLkt7nezpWwMlxkynL26JH5eY',
```

Comment:
Just received an email by Google saying that this API key shouldn't be publicly accessible, can we make sure it is revoked asap?